### PR TITLE
Loading Lobby Art

### PR DESCRIPTION
**Pull request by blackcrystall**
# About the pull request

Loading around

# New icon
(animated, look in vsc/native editor for more info)
image

# Changelog

:cl: BlackCrystalic
qol: if server take too long to pre init all before lobby art, players will see lobbyart "loading"
/:cl:

### CODE REVIEW

**Comment by blackdragonTOW** [comment]
Hello! Please remember to attach screenshots of your altered sprites in the PR description!!

File: maps/map_files/generic/Admin_level.dmm
Comment (Nanu308) [inline maps/map_files/generic/Admin_level.dmm:1]: What's this for?

**Review by Nanu308 — DISMISSED** [review]

**Comment by harryob** [comment]
> Hello! Please remember to attach screenshots of your altered sprites in the PR description!!

this isn't a requirement, image diff bot exists for that

**Review by harryob — APPROVED** [review]

**Review by nauticall — APPROVED** [review]
shweet

**Pull request merged**